cmd/kube-proxy/app/server_others.go

```diff
@@ -247,20 +247,19 @@ func getProxyMode(proxyMode string, iptver iptables.IPTablesVersioner, kcompat i
 	if proxyMode == proxyModeUserspace {
 		return proxyModeUserspace
 	}
-
-	if len(proxyMode) > 0 && proxyMode == proxyModeIPTables {
+	if proxyMode == proxyModeIPTables {
 		return tryIPTablesProxy(iptver, kcompat)
 	}
-
-	if utilfeature.DefaultFeatureGate.Enabled(features.SupportIPVSProxyMode) {
-		if proxyMode == proxyModeIPVS {
+	if proxyMode == proxyModeIPVS {
```

Comment:
Any chance you can refactor these options with a switch statement?

Reply:
yes, updated. @cmluciano  PTAL